.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
+dist: xenial
```

Comment:
Why did we make this change?

Reply:
I was wondering this initially. I figured it was because Python 3.7 was installed by default on this image version. https://docs.travis-ci.com/user/reference/xenial/#python-support

Reply:
Yea, previously we used trusty. But with 3. everyone needs to use a workaround to get it to work.
I found the answer here https://github.com/travis-ci/travis-ci/issues/9815 and https://github.com/travis-ci/travis-ci/issues/9069#issuecomment-425720905